selfdrive/test/process_replay/test_processes.py

```diff
@@ -53,6 +55,25 @@
 BASE_URL = "https://commadataci.blob.core.windows.net/openpilotci/"
 REF_COMMIT_FN = os.path.join(PROC_REPLAY_DIR, "ref_commit")
 
+def run_test_process(data):
```

Comment:
```suggestion

def run_test_process(data):
```
I'd just make sure there's two lines before and after function declarations